code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -8,7 +8,7 @@ LINEN BINS
 	name = "bedsheet"
 	desc = "A surprisingly soft linen bedsheet."
 	description_info = "Click to roll and unroll. Alt-click to fold and unfold. Drag and drop to pick up."
-	description_fluff = "It seems like you can equip it in your backpack slot..."
+	description_info  = "It seems like you can equip it in your backpack slot."
```

Comment:
These two lines overwrite each other, I think? So only the last one stays alive. So if you want, do this:
```dm
	description_info = "Click to roll and unroll. Alt-click to fold and unfold. Drag and drop to pick up.<br>It seems like you can equip it in your backpack slot."
```